detox.config.js

```diff
@@ -35,21 +35,32 @@ function findCurrentDeploymentTarget() {
 
 const iPhoneSimulatorDevice = 'iPhone 11 Pro'
 const currentDeploymentTarget = findCurrentDeploymentTarget()
+const codeSigningDisabled = process.env.CODE_SIGNING_DISABLED === 'true'
 
 /**
  * @param {Configuration} configuration
  * @returns {string}
  */
 function generateBuildCommand(configuration) {
-	return [
+	const baseCommand = [
 		'xcodebuild',
 		'-workspace ios/AllAboutOlaf.xcworkspace',
 		'-scheme AllAboutOlaf',
 		`-configuration ${configuration}`,
 		`-destination 'platform=iOS Simulator,name=${iPhoneSimulatorDevice},OS=${currentDeploymentTarget}'`,
 		'-derivedDataPath ios/build',
 		'build',
-	].join(' ')
+	]
+
+	if (codeSigningDisabled) {
+		baseCommand.push(
```

Comment:
Is it OK for these strings to come at the end of the command? They're not env vars, are they?

Reply:
Should be fine to have them at the end as they're just individual build settings that control and change the way a target is built. Alternatively, we could try to configure code signing settings through fastlane https://docs.fastlane.tools/actions/update_code_signing_settings/ 

I'm wondering how the cached jsbundle is being referenced but we exit out with there being no jsbundle now. I'm wondering if we would issues making an archive build that can be installed with these build settings, but I don't see how we'd have impacted the jsbundle creation or referencing through these changes.